GitHub/README.md

```diff
@@ -10,7 +10,7 @@ In all application and library repositories, we use the `master` branch for prod
 
 When making a change in any repository (whether developing a new feature or fixing a bug), our convention is to create a new branch with the name `<GitHub username>/<short description>` - e.g., `jebeck/new-signup-flow` or `jebeck/bugfix-form-validation`. The backend team occasionally uses sprint-based branch names - e.g., `jh-bate/sprint25` or `jh-bate/sprint25-clinic-roles`.
 
-All significant changes should be made through a pull request that is approved by another Tidepool developer through a process of code review. We often use LGTM "looks good to me" as our approval sign-off at the conclusion of a review and/or one or more rounds of feedback and changes. One of the following (or any similarly unambiguous) emoji is also acceptable as sign-off: 🚢, 👍, or the GitHub ["ship it squirrel"](https://www.quora.com/GitHub/What-is-the-significance-of-the-Ship-It-squirrel 'Quora: Ship It Squirrel') (available with `:shipit:` but sadly a custom, not Unicode, emoji that cannot be pasted into this doc for demonstration).
+All significant changes should be made through a pull request that is approved by another Tidepool developer through a process of code review. We often use LGTM "looks good to me" as our approval sign-off at the conclusion of a review and/or one or more rounds of feedback and changes. One of the following (or any similarly unambiguous) emoji is also acceptable as sign-off: :ship: (:ship:), :thumbsup: (:thumbsup:), or the GitHub ["ship it squirrel"](https://www.quora.com/GitHub/What-is-the-significance-of-the-Ship-It-squirrel 'Quora: Ship It Squirrel') :shipit: (:shipit:,:squirrel:).
```

Comment:
You'll want to use some kind of ignore or literal syntax where you actually want the `:ship:` colon-delimited shortcode to show up...